lib/templates/webmanifest.js

```diff
@@ -66,21 +66,42 @@ const renderWebManifest = (name, color) => `
 			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-list.webp",
 			"sizes": "720x1280",
 			"type": "image/webp",
-			"platform": "narrow",
+			"form_factor": "narrow",
 			"label": "See how you're doing across all your courses - all in one place"
 		},
 		{
 			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-page.webp",
 			"sizes": "720x1280",
 			"type": "image/webp",
-			"platform": "narrow",
+			"form_factor": "narrow",
 			"label": "Understand what you need to get to earn a letter grade"
 		},
 		{
 			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-gpa.webp",
 			"sizes": "720x1280",
 			"type": "image/webp",
-			"platform": "narrow",
+			"form_factor": "narrow",
+			"label": "Harness tailored insights and analysis to maximize your GPA"
+		},
+		{
+			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-list-wide.webp",
+			"sizes": "1280x720",
+			"type": "image/webp",
+			"form_factor": "wide",
+			"label": "See how you're doing across all your courses - all in one place"
+		},
+		{
+			"src": "https://static.gradebook.app/sYbR9JXKTI/screenshot-course-page-wide.webp",
```

Comment:
I only see `screenshot-course-wide.webp` in the CDN, was that a typo?

Reply:
It was a typo in the file name! I corrected it so it will match the mobile screenshots.

Reply:
Cool, I dropped `screenshot-course-wide.webp` and added `screenshot-course-page-wide.webp` 👍